### ISSUE TITLE
Example usage

### ISSUE THREAD

**Issue by avivace** [gbdev/rgbobj#9]
To better promote/show off the tool, it would be cool to add some example usage/output

**Comment by Rangi42** [comment]
@ISSOtm Want to handle this one? I'm not sure if you'd want it in the README, man page, `--help` output, or what.

**Comment by ISSOtm** [comment]
I'm thinking the README and/or website would be appropriate. We would surely want to keep it simple, so I'm thinking of a video of assembling the following ROM and running it in an emulator?

```asm
SECTION "Header", ROM0[$100]

EntryPoint:
    jp Main

    ds $150 - @, 0 ; Reserve some space for RGBFIX to fill in the header.

SECTION "Main", ROM0

Main:
    ; Wait for VBlank.
.waitVBlank
    ldh a, [rLY]
    cp 144
    jr nz, .waitVBlank

    ; Move the Nintendo logo left a little.
    ldh a, [rSCX]
    inc a
    ldh [rSCX], a

    ; And keep going!
    jr Main
```

**Comment by avivace** [comment]
Cool @ISSOtm . Should we create a section / a page on the rgbds-www?

**Comment by ISSOtm** [comment]
If it's meant to show off a little, shouldn't we put it on the front page?

**Comment by avivace** [comment]
> If it's meant to show off a little, shouldn't we put it on the front page?

but then front page of a separate website? Is it worth to build a separate website when rgbds-www?

**Comment by ISSOtm** [comment]
Oh, this is rgbobj, not rgbds itself. My apologies, I got confused.

Scratch what I said above about a video, an [ASCIIcast](https://asciinema.org) would probably be a good idea. Not sure where to put it, though.

Something @Rangi42 and I have been considering for rgbds-www, is creating a page showing peripheral tooling support (e.g. editor/IDE syntax highlighting, etc.), maybe seeded with some of the current content of awesome-gbdev. `rgbobj` would have a good place there!

**Comment by avivace** [comment]
> Oh, this is rgbobj, not rgbds itself. My apologies, I got confused.
> 
> Scratch what I said above about a video, an [ASCIIcast](https://asciinema.org) would probably be a good idea. Not sure where to put it, though.
> 
> Something @Rangi42 and I have been considering for rgbds-www, is creating a page showing peripheral tooling support (e.g. editor/IDE syntax highlighting, etc.), maybe seeded with some of the current content of awesome-gbdev. `rgbobj` would have a good place there!

Yes, exactly. What about a Tools -> rgbobj page ? (with tools in the navbar?). Anyway

Anyway, @ISSOtm the snippet you previously posted for rgbds itself could also be a nice addition to the home page, no? (maybe let's move this discussion to an issue on rgbds-www ?)

**Comment by ISSOtm** [comment]
“Tools” sounds a little less broad than what we're aiming for, since it includes things like syntax highlighting. “Integrations”? (“Support” sounds like we're offering some kind of on-call service :P)
